types/src/eip7594.rs

```diff
@@ -16,19 +16,19 @@ use crate::{
 type FieldElementsPerCell = U64;
 type BytesPerCell = Prod<BytesPerFieldElement, FieldElementsPerCell>;
 
+pub type CellIndex = u64;
+pub type RowIndex = u64;
 pub type ColumnIndex = u64;
 pub type Cell = Box<ByteVector<BytesPerCell>>;
 pub type NumberOfColumns = U128;
 
-type DataColumn<P> = ContiguousList<Cell, <P as Preset>::MaxBlobCommitmentsPerBlock>;
-
 pub type KzgCommitmentsInclusionProofDepth = U4;
 
 pub type BlobCommitmentsInclusionProof = ContiguousVector<H256, KzgCommitmentsInclusionProofDepth>;
 
-pub const CUSTODY_REQUIREMENT: u64 = 4;
+pub const CUSTODY_REQUIREMENT: u64 = 1;
 pub const DATA_COLUMN_SIDECAR_SUBNET_COUNT: u64 = 32;
-pub const SAMPLES_PER_SLOT: u64 = 16;
+pub const SAMPLES_PER_SLOT: u64 = 8;
```

Comment:
can you (refer to sources that) explain these constants?

Reply:
I changed based on the specs: https://github.com/ethereum/consensus-specs/blob/v1.5.0-alpha.3/specs/_features/eip7594/das-core.md#custody-setting

Reply:
thanks. Why tho is there no `TARGET_NUMBER_OF_PEERS` specified?

Reply:
not defined here, it should be in `eth2_libp2p` instead, it also got removed in the new specs